runbotics/runbotics-orchestrator-ui/src/src-landing/components/BlogHeadImage/BlogHeadImage.module.scss

```diff
@@ -2,45 +2,48 @@
 
 
 
-
 .image {
-
     position: relative;
-    margin-top: $HEADER_HEIGHT;
+
     width: 100%;
     height: $BACKGROUND-HEIGHT;
-    background: url('/images/banners/blog-header.png') no-repeat;
+
+    margin-top: $HEADER_HEIGHT;
+
     background-position: center;
     background-size: cover;
     isolation: isolate;
     font-size: 250px;
     &::before {
-        content: '';
         position: absolute;
+
         width: 100%;
         height: 100%;
+
         z-index: $header-z-index;
-        background: transparent url('/images/blog/mask.svg');
+        background: #FBB040;

```

Comment:
isn't it in `_palette.scss`? Looks a bit like our primary colour if its not maybe we should use a primary